docs/bug_report.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: "BUG: "
 name: Bug report
+about: Create a report to help us improve
+
```

Comment:
Added additional space since `name` and `about` are displayed in gui where title, labels etc are in issue